pwdtk/settings.py

```diff
@@ -39,6 +39,8 @@ def add_middlewares(
 # PWDTK Passsword lockout after logins with bad password
 # ----------------------------------------------------------
 
+PWDTK_RAISE_MUST_RENEW_CLASSES = ["rest_framework.request.Request", ]
```

Comment:
comment to explain what `PWDTK_RAISE_MUST_RENEW_CLASSES ` means.
It is not obvious for me.

I guess it is something, that only rest requests will raise renew exceptions but not normal requests.

but I only guess this as I once overheard you talking about something like this

Reply:
It is obsolete, i will remove